### PR TITLE
Design/editor update

### PR DESCRIPTION
**Pull request by CodeTheory**
It's finally here! 🎉 

image

@matheuss @sindresorhus @doot0 Can any of you review? 🙌

### CODE REVIEW

File: app/src/renderer/views/post-recording.pug [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
-      progress(max='100', value=50)
-      section.controls
+      header.window-header.is-floating.is-transparent.is-hidden
+        //- Title Bar
+        .title-bar
+          .title-bar__controls
+            img.hide-window(src='/app/static/traffic-light-close.svg')
+            img.minimize-window(src='/app/static/traffic-light-minimize.svg')
+            img(src='/app/static/traffic-light-disabled.svg')
+          span.window__title.txt Preferences
```
Comment (matheuss) [inline app/src/renderer/views/post-recording.pug]: 😅

File: package.json [hunk 2]
```diff
@@ -69,5 +69,7 @@
       "transform-es2015-modules-commonjs"
     ]
   },
-  "dependencies": {}
+  "dependencies": {
+    "moment": "^2.17.0"
```
Comment (matheuss) [inline package.json]: This shouldn't be here – it's already in the other `package.json` 😄

File: app/src/renderer/views/post-recording.pug (hunk 1 above)
Comment (CodeTheory) [inline app/src/renderer/views/post-recording.pug]: Yes probably 🙄

**Comment by CodeTheory** [comment]
@matheuss Please review again. 😄

File: yarn.lock [hunk 3]
```diff
@@ -3226,6 +3226,10 @@ mkdirp@0.5.0:
   dependencies:
     minimist "0.0.8"
 
+moment:
```
Comment (matheuss) [inline yarn.lock]: Needs to go too 😰 Sorry, forgot to mention in the previous review 😓

File: app/static/pause.svg [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<svg fill="currentColor" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path class="inner-icon" d="M6 19h4V5H6v14zm8-14v14h4V5h-4z"/>
+    <!-- <path d="M0 0h24v24H0z" fill="none"/> -->
```
Comment (matheuss) [inline app/static/pause.svg]: Can/should be removed? 🤔

File: app/static/play.svg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<svg fill="currentColor" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path class="inner-icon" d="M8 5v14l11-7z"/>
+    <!-- <path d="M0 0h24v24H0z" fill="none"/> -->
```
Comment (matheuss) [inline app/static/play.svg]: Same

**Comment by CodeTheory** [comment]
Please review again @matheuss 😅

**Pull request merged**